src/assets.rs

```diff
@@ -266,58 +300,32 @@ impl Assets {
         Ok(self.store.revert_version()?)
     }
 
-    /// Create an unverified and nontemporary asset.
+    /// Create an unverified asset.
+    ///
+    /// Sets the asset as verified if it's in the verified set.
```

Comment:
This seems a little confusing, and TBH I don't think we need to say anything about verified status here. If our overall conceptual documentation (e.g. module level docs) is good, then it should be understood that an asset will only be verified if you've loaded it from a verified lirbary.